### ISSUE TITLE
Looking for a Moder

### ISSUE THREAD

**Issue by vinicius1222** [Tslat/Advent-Of-Ascension#1759]
someone who knows how to edit mods? It's just that I need someone who can translate a mod, some things are in Chinese and it would be better if they translated into English. In advance, thank you!
http://www.mediafire.com/file/ybhx9kg71ltqzqm/%255B1.7.10%255D%255B%25E6%25B1%2589%25E5%258C%2596%255DNevermine-2.04.jar/file
http://www.mediafire.com/file/0yseikpcby1idus/%255B1.7.10%255D%255B%25E6%25B1%2589%25E5%258C%2596%255DNevermine-2.03.jar/file

**Comment by vinicius1222** [comment]
or someone who has the English version of the mod, in version 2.0.4 at least

**Comment by EatingSilencer** [comment]
Why do you need version 2.0.4 in particular exactly? Even if you want to play 1.7.10 there are versions in 1.7.10 in English that are more up to date than 2.0.4.

**Comment by vinicius1222** [comment]
Because of the thermal system that only exists in the 2.0X versions of the mod. The other recent versions I already have

**Comment by DiamondWalker** [comment]
This... probably isn't the place to ask for this.

I'm really not sure why you want the thermal system, since it didn't add that much to the game and there are plenty of other mods that add similar mechanics to the game, but who am I to judge. There's a 2.0 download on the wiki, but idk what the difference between that and 2.0.4 versions are. Alternatively I believe most of this can be changed with a resource pack that replaces the lang file, but don't quote me on that since these old versions had some questionable design.

**Comment by EatingSilencer** [comment]
You....want the thermal system...?

Anyway if your not able to find a version with English (it definitely exists but we don't seem to have it on curseforge) then you can translate using a resource pack. You don't even need another mod.

**Comment by vinicius1222** [comment]
version 2.0 of the wiki seems to be incomplete, the spawn rate is buggy and the campfire is missing, this is very important to maintain the temperature. 
Well, there is not much to do in this one, I was lucky to find these versions. I will have to deal with this, thank you very much for your help

**Comment by EatingSilencer** [comment]
That's because 2.0 is an out of date version of the mod (though the wiki should still have info on it on the removed features page or similar page, unless your looking at a different wiki?)

Also, if by "lucky to find these versions" you mean lucky to find an AoA download at all, you can find aoa downloads here:
[https://minecraft.curseforge.com/projects/advent-of-ascension-nevermine](https://minecraft.curseforge.com/projects/advent-of-ascension-nevermine)

**Comment by vinicius1222** [comment]
Yes, but it doesn't have the specific versions: 2.0.2 and 2.0.4, these were almost impossible to find. I only found it on a Chinese website

**Comment by AstralScourge** [comment]
Why do you need 2.0.2 or 2.0.4 just update the thermal system is bad anyway
